id_ID/ch2-id.md

```diff
@@ -23,30 +23,30 @@ Kami berharap Little Carrot memiliki kemampuan **bergerak secara otonom**. Meski
+"Penempatan" dan "pembangunan peta", dapat dilihat sebagai persepsi dua arah: ke dalam dan ke luar. Sebagai robot yang baik, _Little Carrot_ tidak perlu memahami **negara** (lokasinya) sendiri, tetapi juga lingkungan eksternalnya (peta). Tentu saja, ada banyak cara untuk mengatasi kedua masalah ini. Misalnya, kita bisa meletakkan rel pemandu di lantai ruangan, atau spidol stick seperti gambar yang berisi kode QR di dinding, atau memasang perangkat posisi radio di atas meja. Jika Anda berada di luar ruangan, Anda juga dapat memasang komponen penempatan (seperti yang ada pada ponsel atau mobil) di kepala _Little Carrot_. Dengan perangkat ini, dapatkah kita mengklaim bahwa masalah penentuan penempatan sudah terpecahkan? Mari kita kategorikan sensor ini (see \ autoref {fig: sensors}) menjadi dua kelas.
 
-![sensors](/resources/whatIsSLAM/sensors.jpg)
+![sensor](/resources/whatIsSLAM/sensors.jpg)
 
-The first class are **non-intrusive** sensors which are completely self-contained inside a robot, such as wheel encoders, cameras, laser scanners, etc. They do not assume an cooperative environment. The other class are **intrusive** sensors depending on a prepared environment, such as the above mentioned guiding rails, QR code stickers, etc. Intrusive sensors can usually locate a robot directly, solving the positioning problem in a simple and effective manner. However, since they require changes to the environment, the scope of use is often limited. For example, what if there is no GPS signal, or guiding rails cannot be laid? What should we do in those cases?
+Kelas pertama adalah sensor **non-intrusif** milik robot sendiri, seperti pengatur roda, kamera, pemindai laser, dll. Mereka tidak menganggap lingkungannya kooperatif. Kelas yang lain adalah sensor **intrusive** tergantung pada lingkungan yang dipersiapkan, seperti rel pemandu yang disebutkan di atas, stiker QR code, dll. Sensor _intrusive_ biasanya dapat menemukan robot secara langsung, menyelesaikan masalah penentuan posisi dengan cara yang sederhana dan efektif. Namun, karena membutuhkan perubahan lingkungan, ruang lingkup penggunaannya seringkali terbatas. Misalnya, bagaimana jika tidak ada sinyal GPS, atau rel yang sudah dibangun tidak bisa diletakkan? Apa yang harus kita lakukan pada kasus-kasus tersebut?
```

Comment:
- First, **non-intrusive** is translated to **non-instrusif**, how come **instrusive** is translated to **instrusive** but not **instrusif**?

Reply:
Sorry, its typo. I'ww change it.
Thanks for correction